app/src/main/java/org/oppia/android/app/options/LanguageSelectionViewModel.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.android.app.options
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.MutableLiveData
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+/**
+ * Language list view model for the recycler view in [AppLanguageFragment] and
+ * [AudioLanguageFragment].
+ */
+@FragmentScope
+class LanguageSelectionViewModel @Inject constructor(
+  val activity: AppCompatActivity
+) : ObservableViewModel() {
+
+  val selectedLanguage = MutableLiveData<String>()
+
+  private val appLanguagesList = listOf<LanguageItemViewModel>(
+    LanguageItemViewModel("English"),
+    LanguageItemViewModel("French"),
+    LanguageItemViewModel("Hindi"),
+    LanguageItemViewModel("Chinese")
+  )
```

Comment:
@rt4914 do we have tracking issues to update this & the list below to properly match the list of languages available in the topic playing/that the app is translated to?

Reply:
Yes, https://github.com/oppia/oppia-android/issues/19 and https://github.com/oppia/oppia-android/issues/20